packages/react/src/useTable.tsx

```diff
@@ -205,8 +228,25 @@ export const useTable = <
         },
         sortable: "sortable" in field && field.sortable,
         relatedField,
-      } as TableColumn;
+        isCustomCell: false,
+      } satisfies RecordTableColumnValue;
     });
+
+    if (options?.columns) {
+      // Add custom cell columns
+      for (const custom of options.columns) {
```

Comment:
I think this should be for (const column of options.columns) for more accurate naming